src/graphql/schema.js

```diff
@@ -83,9 +93,50 @@ const UserType = new GraphQLObjectType({
   }),
 });
 
+const InteractionType = new GraphQLObjectType({
+  name: "Interaction",
+  fields: () => ({
+    id: {
+      type: GraphQLID,
+    },
+    name: {
+      type: GraphQLString,
+    },
+    description: {
+      type: GraphQLString,
+    },
+    notes: {
+      type: GraphQLString,
+    },
+    tags: {
+      type: GraphQLString,
```

Comment:
Same, list of strings:
```suggestion
      type: new GraphQLList(GraphQLString),
```

Reply:
Woops--error on not adding changes to latest commit. Fixed.